controller/pages/api/util/pewpew.ts

```diff
@@ -215,3 +227,13 @@ export async function deletePewPew (query: Partial<Record<string, string | strin
     throw error;
   }
 }
+
+export function getPewPewVersionInFile (): string {
+  try {
+    const latestVersion: string = latestPewPewInFile.version;
```

Comment:
We should load this from S3 and then cache it in a global variable on first load. If the file doesn't exist we can set it to `unknown` which should be a `const` that we can check for our yaml validator check.

We then need to add unit and integration tests to validate that the file is read See `PPaasS3File` in common to use to read and write.